czkawka_cli/src/commands.rs

```diff
@@ -0,0 +1,192 @@
+    pub excluded_directories: Vec<PathBuf>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct ExludedItems {
+    #[structopt(short = "E", long, parse(from_os_str), help = "Exluded item(s)", long_help = "List of excluded item(s) which contains * wildcard(may be slow, so use -e where possible)")]
+    pub excluded_items: Vec<PathBuf>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct AllowedExtensions {
+    #[structopt(
+        short = "x",
+        long,
+        help = "Allowed file extension(s)",
+        long_help = "List of checked files with provided extension(s). There are also helpful macros which allow to easy use a typcal extensions like:\nIMAGE(\"jpg,kra,gif,png,bmp,tiff,webp,hdr,svg\"),\nTEXT(\"txt,doc,docx,odt,rtf\"),\nVIDEO(\"mp4,flv,mkv,webm,vob,ogv,gifv,avi,mov,wmv,mpg,m4v,m4p,mpeg,3gp\") or\nMUSIC(\"mp3,flac,ogg,tta,wma,webm\")"
+    )]
+    pub allowed_extensions: Vec<String>,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct NotRecursive {
+    #[structopt(short = "R", long, help = "Prevents from recursive check of folders")]
+    pub not_recursive: bool,
+}
+
+#[derive(Debug, StructOpt)]
+pub struct FileToSave {
+    #[structopt(short, long, value_name = "file-name", help = "Saves the results into the file")]
+    pub file_to_save: Option<PathBuf>,
+}
+
+impl FileToSave {
+    pub fn file_name(&self) -> Option<&str> {
+        if let Some(file_name) = &self.file_to_save {
+            return file_name.to_str();
+        }
+
+        None
+    }
+}
+
+fn parse_checking_method(src: &str) -> Result<CheckingMethod, &'static str> {
+    match src.to_ascii_lowercase().as_str() {
+        "size" => Ok(CheckingMethod::Size),
+        "hash" => Ok(CheckingMethod::Hash),
+        "hashmb" => Ok(CheckingMethod::HashMB),
+        _ => Err("Couldn't parse the search method (allowed: SIZE, HASH, HASHMB)"),
+    }
+}
+
+fn parse_delete_method(src: &str) -> Result<DeleteMethod, &'static str> {
+    match src.to_ascii_lowercase().as_str() {
+        "aen" => Ok(DeleteMethod::AllExceptNewest),
+        "aeo" => Ok(DeleteMethod::AllExceptOldest),
+        "on" => Ok(DeleteMethod::OneNewest),
+        "oo" => Ok(DeleteMethod::OneOldest),
+        _ => Err("Couldn't parse the delete method (allowed: AEN, AEO, ON, OO)"),
+    }
+}
+
+fn parse_min_size(src: &str) -> Result<u64, String> {
+    match src.parse::<u64>() {
+        Ok(min_size) => {
+            if min_size > 0 {
+                Ok(min_size)
+            } else {
+                Err("Minimum file size must be at least 1 byte".to_string())
+            }
+        }
+        Err(e) => Err(e.to_string()),
+    }
+}
+
+static HELP_MESSAGE: &str = "Prints help information (--help will give more information)";
+
+const HELP_TEMPLATE: &str = r#"
+{bin} {version}
+
+USAGE:
+    {usage} [SCFLAGS] [SCOPTIONS]
+
+FLAGS:
+{flags}
+
+SUBCOMMANDS:
+{subcommands}
+
+    use {usage} -h to get more info about a specific tool
```

Comment:
This looks in code that
```
 use czkawka_cli <SUBCOMMAND> -h to get more info about a specific tool
```
but quotes, brackets or something similar should be added to make text easier to read
```
use "czkawka_cli <SUBCOMMAND> -h" to get more info about a specific tool
use [czkawka_cli <SUBCOMMAND> -h] to get more info about a specific tool
use `czkawka_cli <SUBCOMMAND> -h` to get more info about a specific tool
```

Reply:
I will add this:
```suggestion
    try "{usage} -h" to get more info about a specific tool
```